blocks/columns/columns.css

```diff
@@ -188,6 +267,47 @@
   }  
 }
 
+@media (768px <= width <= 1024px) {
+  .columns.columns-transform-teaser .columns-img-col {
+    width: 50%;
+    right: -65px;
+  }
+}
+
+@media (width < 768px) {
```

Comment:
This is why we need to start mobile first. Can we not simplify by using
`width <= 1024px` & `width <= 768px`?

Reply:
When using like this "width <= 1024px & width <= 768px"
its showing lint errors